Curse/main.swift

```diff
@@ -9,3 +9,95 @@ import Foundation
 
 print("Hello, World!")
 
+// 10 вариантов Dictionary
+
+let colorWarm : [Int:String] = [0 : "red", 1 : "orange", 2 : "yellow"]
```

Comment:
если тип не требуется, его писать не нужно
```suggestion
let colorWarm = [0 : "red", 1 : "orange", 2 : "yellow"]
```
И придерживайся написания тип: значение, перед двоеточием пробела нет, после один.

Reply:
поправить везде